_includes/neurips/2020/about.neurips.html

```diff
@@ -0,0 +1,515 @@
+          Authors are also encouraged to sign up to review for LXAI, with a sign-up option available upon submission.
+      </p>
+      <p>
+          Submissions will be double-blind peer-reviewed and may be up to two pages including all figures and tables,
+          with an additional page for references submitted as a PDF. The submissions should strictly follow
+          the guidelines provided by the
+          <a target="_blank" href="https://neurips.cc/Conferences/2020/PaperInformation/StyleFiles">NeurIPS 2020 Style Template</a>
+          to avoid the risk of being rejected without consideration of their merits. Submissions must state the research
+          problem, motivation, and technical contribution. Submissions must be self-contained and include all figures,
+          tables, and references. Authors of accepted abstracts will be asked to present their work in a poster session.
+          A few authors will be selected to give 10–15 minute oral presentations.
+      </p>
+
+      <p>
+          The submission deadline is <strong>October 7th, 2020 Anywhere on Earth</strong>.
+          Please note that no extensions will be offered for submissions.
+      </p>
+      <a class="btn_icml20-blue" target="_blank" href="https://cmt3.research.microsoft.com/LXAINeiurIPS2020">Submit Abstract</a>
+    </div>
+
+
+<!--    <hr style="margin-top: 20px;">
+    <div class="user-projects" id="MentoringProgram">
+        <h2>Mentoring Program</h2>
+        <p>
+            LatinX in AI is hosting a new mentoring program alongside our official workshops.
+            The LatinX in AI Mentoring Program requires mentor and mentees to meet once a month, premiering May 1st to the day of the workshop at ICML on July 13th, 2020.
+            On this day, some mentees will be asked to share their experience and the learnings they obtained from the program.
+        </p>
+        <p>
+            Sign-up to be a mentor: <a class="btn_icml20-green" href="https://docs.google.com/forms/d/e/1FAIpQLScaw0QMM9DWMKUxak4Z96AiIUlP-9kVRfnqijtemP1p05tqCA/viewform">
+            Mentor Signup</a>
+        </p>
+        <p>
+            Sign-up to recieve mentoring: <a class="btn_icml20-green" href="https://docs.google.com/forms/d/e/1FAIpQLSd5Mz___3G3jVOxoTbvCPv8SIn_jTpoXzzUxTknO1gjQUuiQA/viewform">
+            Mentee Signup</a>
+        </p>
+        <p>
+            Following the completion of this program, you'll be asked to complete a quick survey about your experience and opportunities for our improvement.
+            Mentors who complete the program and survey are eligible to receive a $25 gift card as a token of appreciation on behalf of LatinX in AI.
+        </p>
+    </div> -->
+
+    <hr style="margin-top: 20px;">
+    <div class="user-projects" id="workshopSponsors" style="color: black">
+      <h2>Sponsors</h2>
+      <p>If you are interested in becoming a sponsor, please
+      <a href="/neurips-2020-sponsor">Sponsor Prospectus</a>
```

Comment:
the link to the sponsor prospectus is broken and should be removed. Instead the sentence should read 'If you are interested in becoming a sponsor, please send us a message to...'